.idea/deployment.xml

```diff
@@ -1,16 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="PublishConfigData" serverName="localhost uber">
-    <serverData>
-      <paths name="localhost uber">
-        <serverdata>
-          <mappings>
-            <mapping local="$PROJECT_DIR$" web="/home/vagrant/magfest" />
-            <mapping local="" />
-          </mappings>
-        </serverdata>
-      </paths>
-    </serverData>
-  </component>
-</project>
-
+  <component name="PublishConfigData" serverName="localhost uber" />
+</project>
```

Comment:
sidenote: anything in .idea in this repo is old and can get deleted

unless folks are opening this project directly in pycharm as part of docker stuff, in which case it can stay.